### PR TITLE
Adding instructions for advanced usage

### PR DESCRIPTION
**Pull request by assarbad**
I also added some remarks regarding the issue with parsing `ls` output.

Additionally `--editor` was mentioned in one place where previously only `$EDITOR` was mentioned.

Happy new year!

(Hint: you can simply look at the rendered README over in my forked repo)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +37,65 @@ to run `git mv old new` on each rename, you can do something like this:
...
+
+```bash
+find -type f -exec renamer {} \;
+```
+
+In order to sidestep this issue, you can employ `xargs` in conjunction with
+`find` like so (`-print` is implied for `find`):
+
+```
+find -type f|xargs renamer --editor vim
```
Comment (mtimkovich) [inline README.md:82]: nit: could you add a space on either side of the pipe here?

File: README.md [hunk 2]
```diff
@@ -36,6 +37,65 @@ to run `git mv old new` on each rename, you can do something like this:
...
+```
+
+The part past `xargs` is the invocation of `renamer` without the file names.
+It exists just to demonstrate how you would pass arguments to `renamer` using
+this method.
+
+If your files contain wonky characters you could also try:
+
+```
+find -type f -print0|xargs -0 renamer --editor vim
```
Comment (mtimkovich) [inline README.md:92]: same for this

**Review by mtimkovich — APPROVED** [review]
Thanks for this! The little differences with piping in different `ls` versions has been causing quite a headache...

I just have some super small style nits.

Happy new year to you too!

File: README.md (hunk 1 above)
Comment (assarbad) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (assarbad) [reply README.md]: Done

**Pull request merged**